api/notd/store/schema.py

```diff
@@ -17,3 +17,18 @@
     sqlalchemy.Column(key='blockHash', name='block_hash', type_=sqlalchemy.Text, nullable=False),
     sqlalchemy.Column(key='blockDate', name='block_date', type_=sqlalchemy.DateTime, nullable=False),
 )
+
+TokenMetadataTable = sqlalchemy.Table(
+    'tbl_token_metadatas',
+    metadata,
+    sqlalchemy.Column(key='tokenMetadataId', name='id', type_=sqlalchemy.Integer, autoincrement=True, primary_key=True, nullable=False),
+    sqlalchemy.Column(key='createdDate', name='created_date', type_=sqlalchemy.DateTime, nullable=False),
+    sqlalchemy.Column(key='updatedDate', name='updated_date', type_=sqlalchemy.DateTime, nullable=False),
+    sqlalchemy.Column(key='registryAddress', name='registry_address', type_=sqlalchemy.Text, nullable=False),
+    sqlalchemy.Column(key='tokenId', name='token_id', type_=sqlalchemy.Integer, nullable=False),
+    sqlalchemy.Column(key='metadatUrl', name='metadata_url', type_=sqlalchemy.Text, nullable=False),
+    sqlalchemy.Column(key='imageUrl', name='image_url', type_=sqlalchemy.TEXT, nullable=False),
```

Comment:
many of these have `nullable=False` but are nullable in the table definition below. please check everything is consistent here.